content/en/blog/posts/2022-02-01-build-release-v0.8.0/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Shipwright v0.8.0 Is Here"
+date: 2022-02-01T21:36:00-04:00
+draft: false
+author: "Enrique Encalada ([@qu1queee](https://github.com/qu1queee))"
+resources:
+- src: "**.{png}"
+  title: "Figure #:counter"
+---
+
+So, you have heard great things about Shipwright last year and you are ready for more. We are starting the year with our [v0.8.0](https://github.com/shipwright-io/build/releases/tag/v0.8.0), and here is a list of the most relevant things you should know.
+
+## Features
+
+The list of features is not long; however I can ensure that they are all interesting!
+
+### Array support in Parameters
+
+We introduced an extension to the **parameter** feature, by allowing users to define parameters in the form of a list. A list can be composed of values from secrets, configmaps or plain values.
+
+Our main driver was the support for [ARGS](https://docs.docker.com/engine/reference/builder/#using-arg-variables) in Dockerfiles. This allows users to further [customize](https://github.com/shipwright-io/sample-go/blob/main/docker-build-with-args/Dockerfile) their builds, by specifying variables that are available to the `RUN` command.
+
+In addition, being able to use primitive resources (_such as secrets and configmaps_) to store key-values, allows users to protect confidential data or to share data when defining parameters values in their `Builds` or `BuildRuns`.
+
+**Note**: For more details on this, please see the [docs](https://github.com/shipwright-io/build/blob/main/docs/buildstrategies.md#strategy-parameters).
+
+### Surfacing Errors in the BuildRun Status
+
+Surfacing errors from different containers can be a challenging task, not because of technicality, but rather the question of the best way to represent the state. In case of _failure_ or _success_ during execution, we surface the state under the `.status` subresource of a `BuildRun`.
+
+![status subresource](image001.png)
+_The photo highlights the .status.failureDetails. subresource of a BuildRun_
```

Comment:
To support visually handicapped people, I prefer to use Markdown formatting given the screenshot "only" contains text.

Reply:
done